controllers/hibp.js

```diff
@@ -36,6 +37,17 @@ async function notify (req, res) {
       throw new Error("Unrecognized breach: " + reqBreachName);
     }
   }
+  // If we don't have a logo of the breach, get the logo.
+  if (!fs.existsSync(`./public/img/logos/${breachAlert.LogoPath}`)) {
+    try {
+      await BreachLogos.downloadLogo(breachAlert.LogoPath);
+    } catch(err) {
+      // Set LogoPath to missing-logo-icon.png if download fails
+      // LogoPath will reset when the server restarts
+      breachAlert.LogoPath = "missing-logo-icon.png";
+      log.error(`Unable to download logo for ${breachAlert.LogoPath}.`);
```

Comment:
Does someone read/monitor these logs so we know there was a logo fetch failure?

Reply:
Oh, also, I think your `log.error()` won't be useful, because if I'm reading it correctly, it will _always_ say "Unable to download logo for missing-logo-icon.png" (vs using the `breachAlert.Name` or something more descriptive).

Reply:
You're totally right. I'm also now wondering if any retries will hit this catch then look for "missing-logo-icon.png" from HIBP on subsequent requests?

Reply:
These errors are thrown up to Sentry, which sends alerts. Do either of you get them? I can add you to the Sentry group if you like.

Also still need to update these `mozlog` calls to have the 2 expected arguments.

Reply:
@groovecoder I don't think I'm getting Sentry emails, but would like to be added if it means I'll see errors.